datafusion/functions/src/string/contains.rs

```diff
@@ -59,7 +64,13 @@ impl Default for ContainsFunc {
 impl ContainsFunc {
     pub fn new() -> Self {
         Self {
-            signature: Signature::string(2, Volatility::Immutable),
```

Comment:
@jayzhan211  / @notfilippo  should we deprecate `Signature::string` and direct people to `Signature::coerceable`?

Reply:
Actually I'm quite confuse why we need to change to coercible(string, string) in this PR, `String(2)` is equivalent to `Coercible(string, string)` in my mind, so I don't quite follow the change at all